src/actions/index.js

```diff
@@ -1,67 +1,116 @@
-import axios from 'axios';
+import axios from "axios";
 
-import { FETCH_CLUBS, GET_PAGES, GET_HEROS, GET_EVENTS, MODAL_OPEN, MODAL_CLOSE } from './actionTypes';
+import {
+  FETCH_CLUBS,
+  GET_PAGES,
+  GET_HEROS,
+  GET_EVENTS,
+  MODAL_OPEN,
+  MODAL_CLOSE,
+} from "./actionTypes";
 
-const api = 'https://dev.huoa.org/wp-json';
+const api = "https://dev.huoa.org/wp-json";
 
 export const getPages = () => async (dispatch) => {
-  if (process.env.NODE_ENV === "production" && navigator.userAgent !== "ReactSnap") {
+  if (
+    process.env.NODE_ENV === "production" &&
+    navigator.userAgent !== "ReactSnap"
```

Comment:
To save some memory space, lets make this a variable after the api constant. Something like `let isProd = !!(process.env.NODE_ENV === "production" && navigator.userAgent !== "ReactSnap")` . So now we could just reference this local variable in this conditional like `if (isProd) {` we'll be able to save some time and space because we wont be making calls to the `process.env` and `navgator.userAgent` methods. Instead we'll reference the boolean that is set by them.